### DISCUSSION TITLE
Embed a GitHub gist?

### DISCUSSION THREAD

**Discussion by ghost** [retypeapp/retype#241] [Q&A]
Is it possible?

I tried: 

- Using the Gist's 'embed' `<script>...` link renders a blank page.

- `[!embed](...)` doesn't work for the gist's shareable link.

Any ideas?

**Comment by geoffreymcgill** [comment]
Thanks for the report. We will take a look and see if there is a way to make this work right now.

I also have a feature I would like to implement to support this type of embedding in a cleaner way by just including the link to the gist.
